include/seastar/http/routes.hh

```diff
@@ -194,7 +194,7 @@ public:
      * std::rethrow_exception(eptr);
      * and catch only the exception it handles
      */
-    exception_handler_id register_exeption_handler(exception_handler_fun fun) {
+    exception_handler_id register_exception_handler(exception_handler_fun fun) {
```

Comment:
Right. @amnonh FYI

Reply:
Do you want that to be started in this PR, or in a smaller PR dedicated to introducing the API change?

For API breaks, I tend to recommend splitting them into a distinct PR as it's nice to have "harmless commits" separate from "major commits".

For a PR that just deprecates w/o breaking, that doesn't _have_ to be in a distinct PR, although it's perhaps nice because the smaller PR can be pointed to in the release notes and the changes are easier to read/review.